instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerAttributesExtractorTest.java

```diff
@@ -120,7 +115,6 @@ void normal() {
     assertThat(attributes.build())
         .containsOnly(
             entry(SemanticAttributes.HTTP_METHOD, "POST"),
-            entry(SemanticAttributes.HTTP_URL, "http://github.com"),
             entry(SemanticAttributes.HTTP_TARGET, "github.com"),
```

Comment:
I suspect I fudged - this test case should probably be changed to something that looks more normal, e.g. `/repositories/1`

Reply:
fixed 😄